README.md

```diff
@@ -18,16 +18,22 @@ The following instruction assume your are using Python 2 and an Ubuntu installat
 
 ## Dependencies
 
-Pymavlink requires two dependencies :
+Pymavlink requires 3 dependencies :
```

Comment:
You only list two... and on Linux python-dev (e.g.) is also a dependency due to the mavnative stuff.